Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM golang:1.4
+FROM golang:1.7
 
 ADD . /app/src/github.com/Shopify/toxiproxy
-RUN cd /app/src/github.com/Shopify/toxiproxy && GOPATH=/app/src/github.com/Shopify/toxiproxy/Godeps/_workspace:/app go build -ldflags="-X github.com/Shopify/toxiproxy.Version $(cat VERSION)" -o /app/toxiproxy ./cmd
+
+ENV GOPATH /app:$GOPATH
+ENV PATH $PATH:/app
+WORKDIR /app/src/github.com/Shopify/toxiproxy
+RUN go build -ldflags="-X github.com/Shopify/toxiproxy.Version=$(cat VERSION)" -o /app/toxiproxy ./cmd
+RUN go build -o /app/toxiproxy-cli ./cli
```

Comment:
Hey Sirupsen (love logrus man :D),

You're right, too much layers, not necessary.
I've updated the Dockerfile.